cds_ils/importer/providers/safari/rules/document.py

```diff
@@ -34,7 +34,7 @@ def agency_code(self, key, value):
     return value
 
 
-@model.over("authors", "(^1001_)|(^7001_)")
+@model.over("authors", "(^1001_)|(^7001_)|(^100#_)|(^700#_)")
```

Comment:
If the "#" a wildcard and if so what's the point of the first two options?

Reply:
# is not a wildcard (usually `*` is a wildcard), but a MARC field indicator, see for example: https://cds.cern.ch/record/2782002/export/xm?ln=en (`ind1=", ind2="` etc)